setup/terraform/resources/setup.sh

```diff
@@ -222,6 +222,13 @@ EOF
       chkconfig --add efm
       chown -R root:root /opt/cloudera/cem/${EFM_BASE_NAME}
       sed -i.bak 's#APP_EXT_LIB_DIR=.*#APP_EXT_LIB_DIR=/usr/share/java#' /opt/cloudera/cem/efm/conf/efm.conf
+      if [[ $CEM_MAJOR_VERSION == 2 ]]; then
```

Comment:
As part of the latest changes that I made to the edge2ai deployment, I added support for the installation of any OpenJDK releases available in the public archive. All you need now is to replace the "JAVA_PACKAGE_NAME" variable in the stack with "OPENJDK_VERSION" (e.g. OPENJDK_VERSION=17.0.2).

With this, this section here is no longer necessary.
The latest 7.1.9 published also contain the latest CEM and CEM agents versions.